infra/wca_on_rails/production/rails.tf

```diff
@@ -0,0 +1,265 @@
+    },
+    {
+      name  = "ROOT_URL"
+      value = var.ROOT_URL
+    },
+    {
+      name = "DATABASE_HOST"
+      value = "worldcubeassociation-dot-org.comp2du1hpno.us-west-2.rds.amazonaws.com"
+    },
+    {
+      name = "READ_REPLICA_HOST"
+      value = "readonly-worldcubeassociation-dot-org.comp2du1hpno.us-west-2.rds.amazonaws.com"
+    },
+    {
+      name = "CACHE_REDIS_URL"
+      value = "redis://wca-main-cache.iebvzt.ng.0001.usw2.cache.amazonaws.com:6379"
+    },
+    {
+      name = "SIDEKIQ_REDIS_URL"
+      value = "redis://wca-main-sidekiq.iebvzt.ng.0001.usw2.cache.amazonaws.com:6379"
+    },
+    {
+      name = "STORAGE_AWS_BUCKET"
+      value = aws_s3_bucket.storage-bucket.id
+    },
+    {
+      name = "STORAGE_AWS_REGION"
+      value = var.region
+    },
+    {
+      name = "VAULT_AWS_REGION"
+      value = var.region
+    },
+    {
+      name = "S3_AVATARS_REGION"
+      value = var.region
+    },
+    {
+      name = "DATABASE_AWS_REGION"
+      value = var.region
+    },
+    {
+      name = "DISCOURSE_URL"
+      value = var.DISCOURSE_URL
+    },
+    {
+      name = "S3_AVATARS_BUCKET"
+      value = aws_s3_bucket.avatars.id
+    },
+    {
+      name = "S3_AVATARS_ASSET_HOST"
+      value = "https://avatars.worldcubeassociation.org"
+    },
+    {
+      name = "CDN_AVATARS_DISTRIBUTION_ID"
+      value = "ELNTWW0SE1ZJ"
+    },
+    {
+      name = "DATABASE_WRT_USER"
+      value = var.DATABASE_WRT_USER
+    },
+    {
+      name = "VAULT_ADDR"
+      value = var.VAULT_ADDR
+    },
+    {
+      name = "VAULT_APPLICATION"
+      value = var.VAULT_APPLICATION
+    },
+    {
+      name = "TASK_ROLE",
+      value = aws_iam_role.task_role.name
+    }
+  ]
+  pma_environment = [
+    {
+      name = "PMA_USER_CONFIG_BASE64"
+      value = base64encode(templatefile("../templates/config.user.inc.php.tftpl",
+        { rds_host: "worldcubeassociation-dot-org.comp2du1hpno.us-west-2.rds.amazonaws.com",
+          rds_replica_host: "readonly-worldcubeassociation-dot-org.comp2du1hpno.us-west-2.rds.amazonaws.com" }))
+    }
+  ]
+}
+
+data "aws_iam_policy_document" "task_assume_role_policy" {
+  statement {
+    principals {
+      type        = "Service"
+      identifiers = ["ecs-tasks.amazonaws.com"]
+    }
+
+    actions = ["sts:AssumeRole"]
+  }
+}
+
+resource "aws_iam_role" "task_execution_role" {
+  name               = "${var.name_prefix}-task-execution-role"
+  assume_role_policy = data.aws_iam_policy_document.task_assume_role_policy.json
+}
+
+resource "aws_iam_role_policy_attachment" "task_execution_role_attachment" {
+  role       = aws_iam_role.task_execution_role.name
+  policy_arn = "arn:aws:iam::aws:policy/service-role/AmazonECSTaskExecutionRolePolicy"
+}
+
+resource "aws_iam_role" "task_role" {
+  name               = "${var.name_prefix}-task-role"
+  assume_role_policy = data.aws_iam_policy_document.task_assume_role_policy.json
+}
+
+data "aws_iam_policy_document" "task_policy" {
```

Comment:
Cloudwatch invalidation missing

Reply:
added + the other two missing IAM Policies (backups and rds db connect for RDS IAM Auth)